data/playground/tutorial/unit-2-basic-coding.py

```diff
@@ -0,0 +1,13 @@
+def print_color(color, invert=False):
+    if invert:
+        print(invert_color(color))
+    else:
+        print(color)
+
+
+def invert_color(color):
+    if color == "black":
```

Comment:
Don't we need the reverse case?

Reply:
At least to have a code path for all branches

Reply:
That is what you do in the tutorial. This is just the start code of the tutorial; it doesn't do anything

Reply:
oh I thought all the tutorial steps were in yaml format. Got it.